### DISCUSSION TITLE
MEGA Project!

### DISCUSSION THREAD

**Discussion by Agnyy** [opentoonz/opentoonz#6749] [Stuff - General]
Hello everyone, I create a useful package for Opentoonz, which will be useful for both animation, and for editing or motion graphics. 

This pack is inspired by the pack - https://videohive.net/item/big-pack-of-elements/19888878  "Big Pack of Elements"  
![image](https://user-images.githubusercontent.com/25137502/43181749-4ed8c5b6-8fe7-11e8-9e40-72b939239bbc.png)

I would also like to make a tool that would make it easier to work with the pack. as is done here - https://away.vk.com/away.php 
![image](https://user-images.githubusercontent.com/25137502/43181724-28d122e6-8fe7-11e8-9cb5-4401b809942c.png)
. I myself am not a programmer and if there is anyone willing to help with this, then I will be happy. According to the idea in the RT there is already a similar system, it should be made a separate window and slightly modified. How is this done in the link I attached.

Pak is now being supplemented. But much has already been done. For example, the first section with transitions is filled. It included about 45 transitions.

If there are willing to help with the pack, then I will be VERY lucky. Gland does not steal other people's work.

Here is an approximate list of sections.

1. Transitions
     a. Simple transition
     b. Harisma
     c. Anim.
2. Element
3. Tilte
4. Font (Animate)
5. BG
     a. BG Lines
     b. BG
6. 2D FX
     a. Shot
     b. Fire
     c. Water
     d. Flash
7. Character (Animate)
8. Icon

![image](https://user-images.githubusercontent.com/25137502/43182114-ca47c340-8fe8-11e8-9085-f4baca4231fa.png)

Here is the download link. 
https://agnyy.itch.io/big-pack-for-opentoonz

**Comment by Agnyy** [comment]
I made a short video. - https://youtu.be/s4HVlL6UwnI


![image](https://user-images.githubusercontent.com/25137502/43182373-d39feae8-8fe9-11e8-9e4b-6a27fd3a7cfb.png)

**Comment by RodneyBaker** [comment]
Thanks much for this @Agnyy 
Very much appreciated.

Please drop an announcement in the OpenToonz forum where folks can find it!
It might seem a bit odd to put it in #Announcements because the subtitle reads 'Announcements from the OpenToonz Team'.  That reads a bit to me as... news from the official OpenToonz development team but I think it appropriate given the usefulness of your contribution and the fact you are an important part of the opensource development team.   :)

@shun-iwasawa   Any official thoughts on what should go in the OpenToonz forum's #Announcements channel?

**Comment by shun-iwasawa** [comment]
@Agnyy @RodneyBaker 
Yes, you can post in #Announcement category of the forum if you think it is appropriate.

Well, for now I'm using the term "OpenToonz team" for just indicating that the message is posted by a member of the repository owners. You're right @RodneyBaker, you all are OpenToonz community members (or it can also be said as team!) but I just couldn't come up with any good alternatives.

The announcement from "Official OpenToonz Repository Owner Team in Dwango" will always be posted in #Announcement category in the forum, but the category is open to everyone.

By the way, thank you @Agnyy for launching the nice project.
I'm interested in your thoughts - in which license will you distribute the Pack?

**Comment by Agnyy** [comment]
@shun-iwasawa I do not know much about licenses. I think any license that suits you at any discretion of the author is suitable.

@Rodney I do not know much about forums, but I tried to do what you suggested. https://goo.gl/9cxzSe

**Comment by shun-iwasawa** [comment]
> I think any license that suits you at any discretion of the author is suitable.

Well, if authors can approve, [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) would be ideal from a viewpoint of users, just like [MyPaint brushes](https://github.com/mypaint/mypaint/wiki/Licensing-policy).

**Comment by morevnaproject** [comment]
CC-0 is perfect for re-use.

**Comment by Agnyy** [comment]
Well, let it be СС-0.

**Comment by Agnyy** [comment]
I finished the new department "Element". It consists of two subgroups, here is a video with a part of the demonstration. https://youtu.be/dFeX4Krk_SQ

![image](https://user-images.githubusercontent.com/25137502/43360220-3586b5de-92ba-11e8-94bd-846921d36d3f.png)

 https://youtu.be/dFeX4Krk_SQ

**Comment by Agnyy** [comment]
Hello. I was advised to publish the pack on Gumroad. You can download the package all the same for free if you specify 0. In the price field, I was advised to do this, in case someone wants to donate, that would encourage me more to work on the pack.  https://gumroad.com/l/PackToon

**Comment by Agnyy** [comment]
In the "Element" section, two subgroups were added. 
* Decor
* Comics
![image](https://user-images.githubusercontent.com/25137502/43701969-dd911e66-9960-11e8-853c-3dcec979b9f0.png)

**Comment by Agnyy** [comment]
Gumroad Does not allow to load more than 25 megabytes. Therefore, you will have to wait with the update.  But for now, here is a piece of new additions that will be released soon. https://youtu.be/s9e-g1hnSCA 
![image](https://user-images.githubusercontent.com/25137502/49713767-3a970780-fc5b-11e8-83d7-b01abe099593.png) 
There are several backgrounds in the project.
![image](https://user-images.githubusercontent.com/25137502/49714029-564edd80-fc5c-11e8-8fe9-37fe9e67663d.png)

**Comment by Agnyy** [comment]
Also there were the right icons on the projects.
![image](https://user-images.githubusercontent.com/25137502/49713847-88137480-fc5b-11e8-906a-9f4dc8112a47.png)

**Comment by Agnyy** [comment]
Here is a temporary download link. Then the project can be downloaded elsewhere.
https://mega.nz/#!4F0B3Cyb!70OmLc8A0zB8cQZiRmIkVs2YHLIIBfhwGdFc95VnfoI

**Comment by Agnyy** [comment]
Added new Element "Infographics".
![image](https://user-images.githubusercontent.com/25137502/49713963-096b0700-fc5c-11e8-8d8a-4edfe4814155.png)

**Comment by RodneyBaker** [comment]
Thanks @Agnyy 
More great resources for the OT community!

We need to find some ways to put the spotlight on these.

**Comment by Agnyy** [comment]
New download link.  https://agnyy.itch.io/big-pack-for-opentoonz 
![image](https://user-images.githubusercontent.com/25137502/49781646-97aac000-fd24-11e8-8e62-4649709883f9.png)

**Comment by RodneyBaker** [comment]
Tagging as 'roadmap' in anticipation of discussion during 2019 development cycle on how to best proceed with resources contributed to the Opentoonz community to include how to determine which contributions become proposals for inclusion in the core Opentoonz distribution.

Also, where reasonable and practical, recommend this contribution be considered/reviewed as a complementary resource for use with the v1.3 release of Opentoonz.

**Comment by RodneyBaker** [comment]
Closing this PR pending discussion on what the community wishes to do with contributed resources that aren't added into the build.

Is there a plan to include these in the official release?

**Comment by RodneyBaker** [comment]
Opening for discussion of possible inclusion in v1.4 release.
This resource as well as others can provide nice resources/examples of what can be created with Opentoonz.

**Comment by RodneyBaker** [comment]
Closing but let's remember to revisit for possible inclusion in v1.5
